### ISSUE TITLE
won't let certain blocks be added to placelimiter

### ISSUE THREAD

**Issue by Kalleigh6694** [Poeschl/PlaceLimiter#4]
I am having trouble adding some blocks to PlaceLimiter.  Is there a way to add with item ID #'s in lieu of item names?

**Comment by Poeschl** [comment]
I'm trying to avoid specifing blocks by their ids, because they can change during addition or deletion of mods.

However, I tried it out with the BUILDCRAFTBUILDERS_MACHINEBLOCK in the settings and it turned out that the Quarry has different DataIDs depending on the placement.

This for example

``` yml
 - 'BUILDCRAFTBUILDERS_MACHINEBLOCK:2': 1
 - 'BUILDCRAFTBUILDERS_MACHINEBLOCK:3': 1
 - 'BUILDCRAFTBUILDERS_MACHINEBLOCK:4': 1
 - 'BUILDCRAFTBUILDERS_MACHINEBLOCK:5': 1
```

would allow 1 Quarry placement in each direction
It seems that I need to allow wildcard dataIds like in

``` yml
 - 'BUILDCRAFTBUILDERS_MACHINEBLOCK': 1
```

which will only allow one placement of the `BUILDCRAFTBUILDERS_MACHINEBLOCK` block and ignore the dataId.

Will be included in the next version, but I can't give you an ETA yet.

**Comment by Kalleigh6694** [comment]
Well that sounds GREAT! 

I hope you can get it up and working for those types of blocks :-)

**Comment by Poeschl** [comment]
Note for myself:
if hash collides, equals is called

**Comment by Poeschl** [comment]
Should be fixed with 1.2. Tested it out locally and it worked.

**Comment by Kalleigh6694** [comment]
Weooohooo!

I will try it tonight!

Thanks po!
